modules/power-monitoring-accessing-dashboards-developer.adoc

```diff
@@ -0,0 +1,31 @@
+// Module included in the following assemblies:
+
+// * power_monitoring/visualizing-power-monitoring-metrics.adoc
+
+:_mod-docs-content-type: PROCEDURE
+[id="power-monitoring-accessing-dashboards-developer_{context}"]
+= Accessing {PM-shortname} dashboards as a developer
+
+You can access {PM-shortname} dashboards from the *Developer* perspective of the {product-title} web console.
+
+.Prerequisites
+
+* You have access to the {product-title} web console.
+* You have access to the cluster as a developer or as a user.
+* You have installed the {PM-operator}.
+* You have deployed {PM-kepler} in your cluster.
+* You have enabled monitoring for user-defined projects.
+* You have view permissions for the project that you are viewing the dashboard for.
+
+.Procedure
+
+. In the *Developer* perspective of the web console, go to *Observe* -> *Dashboards*.
+
+. From the *Dashboard* drop-down list, select the {PM-shortname} dashboard you want to see: 
+** *Power Monitoring / Overview*
+** *Power Monitoring / Namespace*
+
```

Comment:
It would be nice to get a one-line description of what each of the dashboard shows.

Reply:
There is already a separate section where the types of dashboards are explained. See:[Power monitoring dashboards overview](https://docs.openshift.com/container-platform/4.15/observability/power_monitoring/visualizing-power-monitoring-metrics.html#power-monitoring-dashboards-overview_visualizing-power-monitoring-metrics). It will be redundant to explain the types of dashboards again. Although, do let me know if you need any changes or specifications in the above mentioned section.